### ISSUE TITLE
Is this compatible with Arcturus Morningstar?

### ISSUE THREAD

**Issue by thereis** [JasonWibbo/HabboSwfOpenSource#4]
The title is self explanatory. I've compiled the project but no success on login.

**Comment by JasonWibbo** [comment]
Need rc4

**Comment by thereis** [comment]
Got it working from another source code. But what I'm trying to achieve it is to run the Tablet/Air version. Is there any chance to compile it? I've navigated through the source files and I only see Interfaces, but no code.

**Comment by JasonWibbo** [comment]
To compile this swf you just need to edit that
https://github.com/JasonWibbo/HabboSwfOpenSource/blob/master/compile.php and launch

**Comment by thereis** [comment]
Yup, I've already compiled it, but what about the air version (HabboTablet.swf)?

**Comment by JasonWibbo** [comment]
I dont have air version

**Comment by thereis** [comment]
I saw the air folder in the project root and thought that it was possible to release it. Thank you!
